### PR TITLE
Create url-param remover filter

### PR DESCRIPTION
**Pull request by androidkotlindev**
Allows removing URL parameters (for example utm_source and utm_medium)

### CODE REVIEW

File: data/filters/url-param-remover.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+title: Remove unneeded URL parameters
+params:
+  - name: url-params
+    description: Parameters to hide
+    type: list
+    default: [ "utm_source", "utm_medium", "utm_campaign", "utm_content" ]
```
Comment (xvello) [inline data/filters/url-param-remover.yaml:6]: These values are already covered by the [AdGuard URL Tracking Protection](https://github.com/AdguardTeam/AdguardFilters/tree/master/TrackParamFilter/sections) list, which is shipped by uBO (although disabled by default). I think we should acknowledge this in the filter description and encourage people to use that list, as they'll benefit from community updates.

They still make good example values, as they are pretty innocuous if enabled by anyone.

**Review by xvello — CHANGES_REQUESTED** [review]
Thanks for your PR @androidkotlindev !

As commented below, I am a bit uneasy about the overlap with AdGuard's list, but happy to accept the filter as it'll be a low-maintenance one, with the following changes.

File: data/filters/url-param-remover.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+tags:
+  - url
+  - params
```
[the remark this answers is not in the document]
Comment (androidkotlindev) [reply data/filters/url-param-remover.yaml]: Resolved with a898323

**Comment by xvello** [comment]
Thanks for the updates! Please ping me when you are ready to merge. 

If that's easy for you, I'd appreciate if you could squash everything in one commit. I'm not sure you'll keep attribution if I squash&merge from github. If you're not used to it, don't sweat it.

**Comment by androidkotlindev** [comment]
@xvello This can now be merged!

**Review by xvello — APPROVED** [review]
:+1:

**Pull request merged**